client/components/App.jsx

```diff
@@ -1,45 +1,49 @@
 import React from 'react';
+import styled from 'styled-components';
 import MovieInfo from './MovieInfo.jsx';
 import MoviePoster from './MoviePoster.jsx';
 
+// styled components below
+const Container = styled.section`
+  background: #424242;
+  height: 80vh;
+  width: 36vw;
+`;
+const PosterWrapper = styled.div`
+  margin-top: 0;
+  margin-left: 0;
+`;
+
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      movieId: 1,
+      movieId: 42,
       poster: '',
       movieInfo: null,
     };
   }
 
   // fetch movie poster on mount
   componentDidMount() {
-    this.getMoviePoster();
+    this.getMoviePoster(this.state.movieId);
     this.getMovieInfo(this.state.movieId);
   }
   // get poster image associated with selected movie
-  getMoviePoster() {
-    fetch('/movies/poster')
+  getMoviePoster(id) {
+    fetch(`movies/poster?movieID=${id}`)
       .then(res => res.json())
```

Comment:
Is this fetch function working, even without the full http://localhost URL path? That's really cool, and new info to me if so. Is that because the url is to the same server that's serving up the page itself?

Reply:
Yes, this fetch is working properly and I believe what you are saying is correct.